frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -56,12 +58,21 @@ export const UnsubscribeSurveyModal = ({
             }
         >
             <div className="flex flex-col gap-3.5">
-                {unsubscribeError && (
+                {unsubscribeError ? (
                     <LemonBanner type="error">
                         <p>
                             {unsubscribeError.detail} {unsubscribeError.link}
                         </p>
                     </LemonBanner>
+                ) : (
+                    <LemonBanner type="warning">
+                        <p>
+                            We will attempt to pay any open invoices associated with your account when you unsubscribe.{' '}
```

Comment:
```suggestion
                            Your invoice will be billed immediately.{' '}
```

Yes, people can have more than one. But that's an edge case I don't think we need to cover in the standard UI.